prep-html-script-tag/prep-html-script-tag-quiz-notes.md

```diff
@@ -0,0 +1,37 @@
+# prep-html-script-tag-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What tag is needed to load JavaScript scripts into an HTML document?
+
+- How do you use a script tag to write JavaScript directly in the HTML document?
+
+- How do you use a script tag to load an external JavaScript file?
+
+## Notes
+
+All student notes should be written here.
+
```

Comment:
` The script tag must be entered in the body of the HTML file and you must use an opening and closing script tag. `
` To load an external Javascript file, you must use an src attribute that points to the file or site you want to link`